### PR TITLE
luminous: build/ops: rpm: Build with lttng on openSUSE

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/39332

### CODE REVIEW

**Comment by smithfarm** [comment]
Oops, in Leap 42.3, there is still no lttng-ust for ppc64le. Dropping the second commit.

**Comment by smithfarm** [comment]
@tserong Can you have another look? I am updating the downstream spec file to look like this, too.

File: ceph.spec.in [hunk 1]
```diff
@@ -30,15 +24,24 @@
 %bcond_with lowmem_builder
```
Comment (tserong) [inline ceph.spec.in]: I've just noticed the above line should probably be removed (given lowmem_builder is set one way or another in the blocks below)

Comment (smithfarm) [reply ceph.spec.in]: @tserong Good catch! Done.

**Comment by tserong** [comment]
LGTM (I hope, these bcond things hurt my brain every damn time, no matter how many times I go through them)

**Comment by yuriw** [comment]
[wip-yuri2-testing-2019-07-15-1956-luminous](https://trello.com/c/5QdUgJyi/667-wip-yuri2-testing-2019-07-15-1956-luminous)

**Pull request merged**